ansible/server-state-playbooks/www-dev/templates/www-dev.openmicroscopy.org.conf.j2

```diff
@@ -14,8 +14,17 @@ server {
         location / {
                 # First attempt to serve request as file, then
                 # as directory, then fall back to displaying a 404.
-                try_files $uri $uri/ /index.html;
+                try_files $uri $uri/ =404;
                 # Uncomment to enable naxsi on this location
                 # include /etc/nginx/naxsi.rules
         }
+
+        location /community {
+                proxy_set_header Host www.openmicroscopy.org;
+                proxy_pass $scheme://134.36.65.239$request_uri/;
```

Comment:
It was in the nginx.com docs I was following, so happy to have advice if you think it's wrong.